config.yaml

```diff
@@ -1,5 +1,7 @@
 extensions:
   health_check:
+    endpoint:localhost:13424
+    path: /health/data
```

Comment:
The formatting is off here and why is this configs path set to `/health/data`?

Reply:
Thanks. I pushed this change by mistake. Reverted.

Reply:
I pushed this change by mistake. I have reverted now. Thanks.